src/main/webapp/javascript/user-page.js

```diff
@@ -30,10 +29,38 @@ async function loadUserRecipes() {
           elementsToAddToImageDiv.forEach(elem => imageDivs[i].appendChild(elem));
         }
       }
+
+      // functionality of delete button
```

Comment:
I think this delete logic could merit its own method, and then only call it in loadUserRecipes(). This could help in the future when you add functionality to handle adding to cookbook or planner, where having all the functionality in one method could become difficult to understand.